packages/web-components/fast-foundation/src/menu-item/menu-item.template.ts

```diff
@@ -22,5 +25,25 @@ export const MenuItemTemplate = html<MenuItem>`
             <slot></slot>
         </span>
         ${endTemplate}
```

Comment:
Do we want some way to include an icon if a menu item has a sub menu? I think this would be different then `endTemplate`

Reply:
Sounds good, but I'll do that in another pr.  Starting on that now.